### PR TITLE
TTT-1922-Implement skeleton for functional/integration tests

### PR DESCRIPTION
**Pull request by RalucaBogdan**
#### Where should the reviewer start?
/component-test/src/test/scala/cts/HealthCheckSpec.scala
#### What tests were added?
Example for healtcheck enpoint test
#### Outstanding questions?
None
#### Definition of Done:
- [ ] Is there appropriate test coverage?
- [ ] Does the confluence need to be updated?
- [ ] Does this add new dependencies? If so, have the healthchecks been updated?
- [ ] Any Deployment changes we need to be aware of? New infrastructure? Changes to existing components?
- [ ] Is there appropriate logging included?
- [ ] Does it imply any kind of data migration? If so what are the steps for that?
- [ ] Is the Documentation been updated, if needed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+Testing
+
+Requirements:
+
+wiremock
+
+For component tests, first start your local wiremock by navigating in your terminal to the path where the wiremock jar is present and run the following command:
+
+java -jar wiremock-standalone-2.7.1.jar --port=19999 --root-dir=/path/to/project/travel-hotel-search/component-test/src/test/resources/wiremock/
```
Comment (claudiasion) [inline README.md]: why travel-hotel-search in the root-dir?

Comment (RalucaBogdan) [reply README.md]: Was a copy-paste :)

File: component-test/src/main/scala/cts/TestConfig.scala [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package cts
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+import scala.language.postfixOps
+
+
+object TestConfig {
+  val IsIntegratedEnv: Boolean = System.getenv("ENVIRONMENT") == "preprod"
```
Comment (adicontiu) [inline component-test/src/main/scala/cts/TestConfig.scala]: Maybe rename to isIntegrationEnv :)

File: component-test/src/main/scala/cts/components/Expedia.scala [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package cts.components
+
+import cts.TestConfig.ServiceBaseUrl
+
+object Expedia {
```
Comment (adicontiu) [inline component-test/src/main/scala/cts/components/Expedia.scala]: Just keep in mind that the name of the service will be different, travel-package-search-service.

**Pull request merged**